packages/plugin-javascript/index.js

```diff
@@ -74,6 +74,10 @@ export default {
   name: 'JavaScript',
 
   resolve(path, [target]) {
+    if (target.startsWith('node:')) {
+      target = target.substring(5);
```

Comment:
Today I learned about `substring`. Nice one

Reply:
I came across this after creating the PR. Not sure if there's a preferred way of doing this.

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-string-slice.md

Reply:
Personal preference I would say. `substring` seems to be less well known, but don't worry about it.